### PR TITLE
[one-cmds] Revise the way command schema is interpreted

### PR DESCRIPTION
**Pull request by mhs4670go**
This commit revises the way command schema is interpreted.

### as-is

```bash
/usr/share/one/backends/command/${CATEGORY}/${TARGET_NAME}.py
# e.g. /usr/share/one/backends/command/codegen/rose.py
# /usr/share/one/backends/command/codegen/sunflower.py
# /usr/share/one/backends/command/profile/rose.py
```

### to-be

```bash
/usr/share/one/backends/command/${BACKEND}/${CATEGORY}.py
# e.g. /usr/share/one/backends/command/tv2/codegen.py
# /usr/share/one/backends/command/tv2/profile.py
```

ONE-DCO-1.0-Signed-off-by: seongwoo <mhs4670go@naver.com>

### CODE REVIEW

File: compiler/one-cmds/tests/one-codegen_005.test [hunk 1]
```diff
@@ -16,33 +16,68 @@
...
+  TARGET_ALREADY_EXIST=false
+fi
+
 # copy dummy-compile to bin folder
 cp dummy-compile ../bin/dummy-compile
+cp ${targetfile} ../target/
 
 # run test
-one-codegen -C ${configfile} -T rose > ${filename}.log 2>&1
+one-codegen -C ${configfile} -T one-codegen_005 > ${filename}.log 2>&1
```
Comment (shs-park) [inline compiler/one-cmds/tests/one-codegen_005.test:76]: I have a question.

There are two locations of target defined.
Here `-T one-codegen_005` and ini file `TARGET=one-codegen_005`.
Which one is actutally used? 😅

**Review by shs-park — APPROVED** [review]
I left a question, PTAL
=)

**Review by seanshpark — APPROVED** [review]
LGTM

File: compiler/one-cmds/tests/one-codegen_005.test (hunk 1 above)
Comment (mhs4670go) [reply compiler/one-cmds/tests/one-codegen_005.test:76]: @shs-park 
Former is going to be used. We would prefer the command line values to one of the configuration file.

Comment (shs-park) [reply compiler/one-cmds/tests/one-codegen_005.test:76]: Okay, thanks!

**Comment by mhs4670go** [comment]
@seanshpark merge please:)

**Pull request merged**